sdk/spring/azure-spring-cloud-stream-binder-servicebus-queue/README.md

```diff
@@ -126,11 +126,90 @@ SessionID | com.azure.spring.integration.servicebus.converter.ServiceBusMessageH
 CorrelationId | com.azure.spring.integration.servicebus.converter.ServiceBusMessageHeaders.CORRELATION_ID | String | N/A
 To | com.azure.spring.integration.servicebus.converter.ServiceBusMessageHeaders.TO | String | N/A
 ReplyToSessionId | com.azure.spring.integration.servicebus.converter.ServiceBusMessageHeaders.REPLY_TO_SESSION_ID | String | N/A
-PartitionKey | com.azure.spring.integration.servicebus.converter.ServiceBusMessageHeaders.PARTITION_KEY | String | N/A
+**PartitionKey** | com.azure.spring.integration.servicebus.converter.ServiceBusMessageHeaders.PARTITION_KEY | String | 1
```

Comment:
We need to add the README content to azure-spring-integration-servicebus and azure-spring-cloud-starter-servicebus